proto/storage/alert.proto

```diff
@@ -17,11 +17,11 @@ message Alert {
         string                  id                  = 1 [(gogoproto.moretags) = "search:\"Deployment ID,store,hidden\" sql:\"index=hash\""];
         string                  name                = 2 [(gogoproto.moretags) = "search:\"Deployment,store\""];
         string                  type                = 4;
-        string                  namespace           = 5 [(gogoproto.moretags) = 'search:"Namespace"'];
```

Comment:
We have same search tag on multiple search fields within alert object. Does this function correctly with bleve? @connorgorman @viswajithiii